motr/setup.h

```diff
@@ -339,6 +339,9 @@ struct m0_reqh_context {
 
 	/** Enable Fault Injection Service */
 	bool                         rc_fis_enabled;
+
+	/** ADDB Record Max record size" */
```

Comment:
/** ADDB Record Max record size in bytes */

Reply:
Corrected. e3b8f9e